docs/_static/css/custom.css

```diff
@@ -0,0 +1,6 @@
+.wy-nav-content{
+    max-width: 100%;
+}
+.rst-content{
+    width: 75%;
+}
```

Comment:
Please add a newline.

Reply:
@m1093782566 Do you want me to add a newline at the end of the file?  
Or is it in between the two styles?

Reply:
At the end of file please :)

Reply:
Done.